StitchCore/Sources/StitchCore/StitchAppClientInfo.swift

```diff
@@ -1,16 +1,37 @@
+/**
+ * A struct providing basic information about a Stitch app client.
+ */
 public struct StitchAppClientInfo {
+    /**
+     * Initializes the `StitchAppClientInfo`.
+     */
     public init(clientAppId: String,
-         dataDirectory: String,
-         localAppName: String,
-         localAppVersion: String) {
+                dataDirectory: String,
+                localAppName: String,
+                localAppVersion: String) {
         self.clientAppId = clientAppId
         self.dataDirectory = dataDirectory
         self.localAppName = localAppName
         self.localAppVersion = localAppVersion
     }
-    
+
+    /**
+     * The client app ID of the Stitch application that this client communicates with.
+     */
     public let clientAppId: String
+
+    /**
+     * The local data directory for any attached instance of mobile MongoDB.
```

Comment:
[nit] This is more generally the data directory Stitch is allowed to use to store any data. Mobile MongoDB will get its data stored here however.